default.nix

```
@@ -54,7 +54,10 @@ rec {
     cargo-ledger --use-prebuilt ${appExe} --hex-next-to-json ledger ${device}
 
     mkdir -p $out/rust-app
-    cp app_${device}.json app.hex $out/rust-app
+    # Create a file to indicate what device this is for
+    touch $out/rust-app/device-${device}
```

This is pure bike-shedding, but it might make it easier to programmatically get the device name?
```suggestion
    echo ${device} > $out/rust-app/device
```

---

Can also put the device in the derivation name, FWIW.

---

Oh this is for the case if the user somehow obtains two tarballs named "release.tar.gz" (instead of "release_nanos.tar.gz").
So the derivation name is irrelevant.